.circleci/config.yml

```diff
@@ -128,6 +128,10 @@ jobs:
           name: Run Unit Tests
           command: |
             make test
+      - run:
+          name: Run Install Tests
+          command: |
+            make test-install
```

Comment:
🔍 note: Ahha! Just as we hoped:

> 🔗 Adding a symbolic link from /home/circleci/.local/bin/slack-test to /home/circleci/.slack/bin/slack

https://app.circleci.com/pipelines/github/slackapi/slack-cli/1216/workflows/f41bd0c2-4edb-4422-88b9-2bd869f702c0/jobs/3607?invite=true#step-106-1097_101

Reply:
🪬 ramble: Compared to:

> 🔗 Adding a symbolic link from /usr/local/bin/slack-test to ~/.slack/bin/slack

https://github.com/slackapi/slack-cli/actions/runs/17590393385/job/49969493179#step:6:34